OlinkAnalyze/R/Olink_anova.R

```diff
@@ -443,9 +454,19 @@ olink_anova_posthoc <- function(df,
     stop("All effect terms must be included in the variable argument or model formula.")
   }
 
+  # Stop if control assays have not been removed
+  ctrl_types <- c("amp_ctrl", "ext_ctrl", "inc_ctrl")
```

Comment:
could this be changed to if any(df$AssayType != "assay")

Reply:
Will change in next version